integration-tests/manager-integration-tests/manager-integration-tests-v2/src/test/java/com/redhat/service/smartevents/integration/tests/v2/steps/Hooks.java

```diff
@@ -60,85 +68,86 @@ public void before(Scenario scenario) {
         context.setStartTime(Instant.now());
     }
 
+    @After
+    public void cleanUp() {
+        boolean disabledCleanup = Boolean.parseBoolean(DISABLE_CLEANUP);
+        boolean disabledFailedCleanup = Boolean.parseBoolean(DISABLE_FAIL_CLEANUP);
+        boolean scenarioFailed = context.getScenario().isFailed();
+        if (disabledCleanup) {
+            context.getScenario().log("Cleanup disabled, all resources stays in the cluster");
+        } else if (disabledFailedCleanup && scenarioFailed) {
+            context.getScenario().log("Failed scenario cleanup disabled and the scenario failed, all resources stays in the cluster");
+        } else {
+            // Delete AWS SQS queues
+            //cleanAwsSQSQueues();
+
+            // Delete Kafka topics and related ACLs
+            //cleanKafkaTopics();
```

Comment:
IMHO those cleanups can be already uncommented so it won't be forgotten once we use SQS and Kafka in tests.

Reply:
Done